signer/src/storage/postgres.rs

```diff
@@ -1549,6 +1549,7 @@ impl super::DbRead for PgStore {
                     ON parent.block_hash = last.parent_hash
                 JOIN bitcoin_blockchain anchor
                     ON anchor.block_hash = parent.bitcoin_anchor
+                    AND anchor.block_height >= $3
```

Comment:
Isn't this redundant, having `bitcoin_blockchain` already filtering for this (in `bitcoin_blockchain_until`)?

Reply:
Yup, removed in [79a2e95](https://github.com/stacks-network/sbtc/pull/1397/commits/79a2e95507fe3912717f7ae3de8c464a55b5bdb1)